cmd/osv-scanner/scan/source/main.go

```diff
@@ -55,7 +55,7 @@ var projectScanFlags = []cli.Flag{
 
 var projectScanExperimentalFlags = []cli.Flag{
```

Comment:
it seems we don't need this variable anymore. We can just move these two into the `projectScanFlags`.